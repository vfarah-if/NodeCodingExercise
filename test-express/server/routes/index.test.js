import router from './index';

describe('index', () => {
    test('should get a rendered response in an expected format', () => {
        console.debug(router);
        expect(router).toBeDefined();
    });    
});
