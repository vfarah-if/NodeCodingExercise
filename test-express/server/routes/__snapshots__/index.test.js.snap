// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index or home page should get the route 1`] = `
"<html>

<head>
  <title>Express</title>
  <link rel=\\"stylesheet\\" href=\\"/stylesheets/style.css\\">
</head>

<body>
  <h1>Express</h1>
  <p>Welcome to Express</p>
</body>

</html>
"
`;
